src/main/scala/com/microsoft/hyperspace/actions/CreateActionBase.scala

```diff
@@ -142,6 +149,22 @@ private[actions] abstract class CreateActionBase(dataManager: IndexDataManager)
         resolvedIndexedColumns)
   }
 
+  private def leafFileInfo(path: Path): Seq[FileInfo] = {
```

Comment:
nit: `leafFilesInfo`? or even `indexFilesInfo` is fine since there are some logics related to index data structure in this function.

Reply:
done, thanks @imback82